charts/janssen-all-in-one/values.yaml

```diff
@@ -538,74 +616,6 @@ nginx-ingress:
-    openidAdditionalAnnotations: { }
-    # -- device-code ingress resource labels. key app is taken
-    deviceCodeLabels: { }
-    # -- device-code ingress resource additional annotations.
-    deviceCodeAdditionalAnnotations: { }
-    # -- Firebase Messaging ingress resource labels. key app is taken
-    firebaseMessagingLabels: { }
-    # -- Firebase Messaging ingress resource additional annotations.
-    firebaseMessagingAdditionalAnnotations: { }
-    # -- uma2 config ingress resource labels. key app is taken
-    uma2ConfigLabels: { }
-    # -- uma2 config ingress resource additional annotations.
-    uma2AdditionalAnnotations: { }
-    # -- webfinger ingress resource labels. key app is taken
-    webfingerLabels: { }
-    # -- webfinger ingress resource additional annotations.
-    webfingerAdditionalAnnotations: { }
-    # -- webdiscovery ingress resource labels. key app is taken
-    webdiscoveryLabels: { }
-    # -- webdiscovery ingress resource additional annotations.
-    webdiscoveryAdditionalAnnotations: { }
-    # -- SCIM config ingress resource labels. key app is taken
-    scimConfigLabels: { }
-    # -- SCIM config ingress resource additional annotations.
-    scimConfigAdditionalAnnotations: { }
-    # -- SCIM config ingress resource labels. key app is taken
-    scimLabels: { }
-    # -- SCIM ingress resource additional annotations.
-    scimAdditionalAnnotations: { }
-    # -- configAPI ingress resource labels. key app is taken
-    configApiLabels: { }
-    # -- ConfigAPI ingress resource additional annotations.
-    configApiAdditionalAnnotations: { }
-    # -- u2f config ingress resource labels. key app is taken
-    u2fConfigLabels: { }
-    # -- u2f config ingress resource additional annotations.
-    u2fAdditionalAnnotations: { }
-    # -- fido2 config ingress resource labels. key app is taken
-    fido2ConfigLabels: { }
-    # -- fido2 config ingress resource additional annotations.
-    fido2ConfigAdditionalAnnotations: { }
-    # -- fido2 ingress resource labels. key app is taken
-    fido2Labels: { }
-    # -- fido2 ingress resource additional annotations.
-    fido2AdditionalAnnotations: { }
-    # -- Auth server ingress resource labels. key app is taken
-    authServerLabels: { }
-    # -- Auth server ingress resource additional annotations.
-    authServerAdditionalAnnotations: { }
-    # -- Casa ingress resource labels. key app is taken
-    casaLabels: { }
-    # -- Casa ingress resource additional annotations.
-    casaAdditionalAnnotations: { }
-    # -- SAML config ingress resource labels. key app is taken
-    samlLabels: { }
-    # -- SAML ingress resource additional annotations.
-    samlAdditionalAnnotations: { }
-    # -- Lock config ingress resource labels. key app is taken
-    lockConfigLabels: {}
-    # -- Lock config ingress resource additional annotations.
-    lockConfigAdditionalAnnotations: {}
-    # -- Lock ingress resource labels. key app is taken
-    lockLabels: {}
```

Comment:
The `lockLabels` and `lockAdditionalAnnotations` should be moved to appropriate location instead of deleting them.